### DISCUSSION TITLE
enviroment not set

### DISCUSSION THREAD

**Discussion by riky177** [nextauthjs/next-auth#2058] [Help]
Hi, I'm learning nextjs and next auth, but I got warning like this in cmd 
```js
https://next-auth.js.org/warnings#nextauth_url NEXTAUTH_URL environment variable not set
```
So i make a file name .env like this 
```js
//that the domain I hosting
NEXTAUTH_URL=https://topremit.vercel.app
```
But I still got that warning, can someone help me ?

**Comment by jpomykala** [comment]
I have the same problem

**Comment by balazsorban44** [comment]
A reproduction would help. 😊

**Reply by jpomykala** [reply]
I'm still trying to solve the issue on my own. I'm not sure if it's a NextAuth, NextJS or my problem :) 
I found an intermediate fix. I added `ENV NEXTAUTH_URL` to my Dockerfile:
![Screenshot 2021-07-27 at 14 57 16](https://user-images.githubusercontent.com/1548104/127157367-8a2ac9cf-c699-4f97-8ea8-67558db7d3ce.png)
